index.html

```diff
@@ -852,10 +856,26 @@
 **`coat_affect_ior`**       | Affect IOR       | `float`  | $ [0, 1]   $ | $ 1         $ | How much to prevent the coat from affecting the relative IOR of layers below
  -->
 
-![](images/coat_0.png width=90% align=right) ![](images/coat_1.png width=90% align=left)
-<div class="shifted-caption">
-    ![Figure [coat]: Coat adds a secondary specular highlight and optional absorption tint](dummy)
-</div>
+
+### TIR
```

Comment:
Since it's only defined (and in any case it will always be defined after the table of contents):
```suggestion
### Total internal reflection
```